src/libraries/Microsoft.Extensions.Hosting.WindowsServices/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP1002</DiagnosticId>
+    <Target>System.ServiceModel.Internals.dll</Target>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP1002</DiagnosticId>
+    <Target>SMDiagnostics.dll</Target>
```

Comment:
Why does this one still need these suppressions? I thought that was fixed with your change?

Reply:
Disregard, this was the first PR that caught the issue with these references.

Reply:
Yup, haven’t updated the repo with the new dependency’s logic yet

Reply:
@joperezr can we please do that for main? Would be great to get the suppression files as clean as possible.